### DISCUSSION TITLE
When you so stupid github think you Are enemy.

### DISCUSSION THREAD

**Discussion by prettytired** [community/community#144121] [New to GitHub]
Impossible to understand anything almoust. Fml. Have good afternoon.

**Comment by ghostinhershell** [comment]
Hey there @prettytired 👋🏾 

First I would like to say welcome to GitHub! I understand that starting with GitHub can feel overwhelming at first. You're not alone—many of us felt the same way when we began. Reaching out in our community was a great first step, we're here to help you navigate and make this easier.

Here are some resources that cater to beginners:
- [🎥 [Video Series] Let's learn git and GitHub together!](https://github.com/orgs/community/discussions/126301)
- [GitHub's "Hello World" Guide](https://docs.github.com/en/get-started/quickstart/hello-world): A step-by-step introduction to GitHub basics.
- [GitHub Skills](https://skills.github.com/), which has hands-on lessons for new users.

Have you taken a look at our Docs? There are a ton of resources in there so I can see how it can be daunting. Here are some things that you can focus on to help get you up and running:
- [About GitHub and Git](https://docs.github.com/en/get-started/start-your-journey/about-github-and-git)
- [Quickstart for repositories](https://docs.github.com/en/repositories/creating-and-managing-repositories/quickstart-for-repositories)
- [Cloning repositories](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) and [Committing changes](https://docs.github.com/en/pull-requests/committing-changes-to-your-project) to your project.
- [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/creating-an-issue)
- [About pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)

Finally, I would say start small. You can create a simple first project to help . For example, you can create a repository to store notes, recipes, or even your ideas. Repos can be used to store more than code! 

Take a look and let us know if you have any additional questions. Cheers!
